napari/layers/base/base.py

```diff
@@ -1545,11 +1456,8 @@ def _update_draw(
         """
         self.scale_factor = scale_factor
 
-        displayed_axes = self._displayed_axes
```

Comment:
This chain of function/property calls was particularly fun. I'm pretty sure this simplifies to just getting the displayed dims (because order first sorts then unsorts the displayed dimensions), but feel free to correct.